### DISCUSSION TITLE
Doesn't censor images on Milovana.com EOS and flash webteases

### DISCUSSION THREAD

**Discussion by AnomalousCabbage** [silveredgold/beta-protection#52] [Q&A] [answered]
Hi,

so, it doesn't censor images on Milovana.com EOS and flash webteases, which are just user created JOI games. (Eos, flash and classic are the 3 webtease formats on the website)

It looks like beta-protection knows they are images because when I right click I see "censor this image" but nothing happens when clicked and the backend doesn't get a "request".

Beta safety also had this issue in a pervious version where images wouldn't censor on EOS teases (though it did work on the flash ones) and when it was later fixed it was now able to censor local images that were opened in the browser (idk if these 2 issues are connected though).

Examples of webtease formats (nsfw):
classic: https://milovana.com/webteases/showtease.php?id=37475 (works)
flash: https://milovana.com/webteases/showtease.php?id=37541 (doesn't work, but new teases are created in eos now anyway)
EOS: https://milovana.com/webteases/showtease.php?id=50879 (doesn't work)

Its just one small website so don't feel pressured to fix any of this but it would be nice if it was.

Thanks!

**Comment by silveredgold** [comment, marked as answer]
Hmmm, that's a tricky one. I'll try and look into it at some point but those teases are effectively an entirely separate web app being hosted inside the "outer" page so I'll need to work out some way to watch the changes in that page without it dragging page performance.

I can't guarantee whether it will be fixable (or how long it might take) but I'll at least try and have a look at options for that.

**Reply by DailyTwo** [reply]
I'm experiencing the same issue. I've noticed the teases being in an iframe and this might cause the issue here. 
If I open images directly (left click -> open image in new tab) the censoring process works as intended.
